lib/index.js

```diff
@@ -27,10 +26,8 @@ var Comscore = module.exports = integration('comScore')
  */
 
 Comscore.prototype.initialize = function() {
```

Comment:
Maybe add a comment here like 'don't load script until a page comes in so that the page comes from loading the script can use the parameters from the first page', just to add a little background as to why the script isn't being loaded in initialize

Reply:
Added 👇